test/test.tags.js

```diff
@@ -1,182 +1,124 @@
 'use strict';
-const code = require('code');
-const lab = exports.lab = require('lab').script();
+const tap = require('tap');
 const setup = require('./setup.test.js');
 
-lab.experiment('tags', { timeout: 5000 }, () => {
-  lab.beforeEach({ timeout: 5000 }, (done) => {
-    setup.withRapptor({}, [], done);
-  });
-  lab.afterEach({ timeout: 5000 }, (done) => {
-    setup.stop(done);
-  });
-  lab.test('can use the tags method to get a list of tags used in the db', (done) => {
-    code.expect(setup.server).to.not.equal(null);
-    const payloads = [
-      {
-        type: 'BankAccount',
-        tags: { currency: 'dollars' },
-        value: 142000000,
-        data: 'liquid Assets only',
-        userId: 'Montgomery Burns'
-      },
-      {
-        type: 'BankAccount',
-        tags: { currency: 'dollars', units: 'cents' },
-        value: 0.15,
-        data: 'liquid assets only',
-        userId: 'Barney'
-      },
-      {
-        type: 'Radish',
-        tags: { animalVegetableMineral: 'vegetable' },
-        value: 1,
-        data: 'radishes are a good source of electrolytes and minerals ',
-        userId: 'user1234'
-      }
-    ];
-    // add a couple of fake metrics:
-    setup.server.inject({
-      method: 'POST',
-      url: '/api/track',
-      payload: payloads[0]
-    }, (err) => {
-      setup.server.inject({
-        method: 'POST',
-        url: '/api/track',
-        payload: payloads[1]
-      }, () => {
-        setup.server.inject({
-          method: 'POST',
-          url: '/api/track',
-          payload: payloads[2]
-        }, () => {
-          setup.server.inject({
-            url: '/api/tags',
-            method: 'GET'
-          }, (response) => {
-            code.expect(response.statusCode).to.equal(200);
-            code.expect(typeof response.result).to.equal('object');
-            code.expect(response.result.length).to.equal(3);
-            code.expect(response.result).to.include('currency');
-            code.expect(response.result).to.include('units');
-            setup.stop(done);
-          });
-        });
-      });
-    });
-  });
-  lab.test('can use the tags method with the optional type parameter', { timeout: 5000 }, (done) => {
-    code.expect(setup.server).to.not.equal(null);
-    const payloads = [
-      {
+tap.beforeEach((done) => {
+  setup.withRapptor({}, [], done);
+});
+tap.afterEach((done) => {
+  setup.stop(done);
+});
+tap.test('can use the tags method to get a list of tags used in the db', (t) => {
+  t.notEqual(setup.server, null);
+  const payloads = [
```

Comment:
there were 2 lab.experiments here requiring 2 different beforeEach/afterEach methods, so I put one of them in a new file test.tags2.js so that tap would have no problem running each one.  I tried to wedge them both into the same file with async.auto, but tap didn't seem to like it.